whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -22170,9 +22170,119 @@
         }
       ]
     },
-    "430": {
-      "ignored": true,
-      "TODO": "map, alternativeForm a Work (like a structured altLabel)"
+    "430" : {
+      "aboutEntity": "?thing",
+      "addLink": "hasVariant",
+      "include": ["i2-nonfiling"],
+      "resourceType": "Work",
+      "pendingResources": {
+        "_:title": {
+          "addLink": "hasTitle", "resourceType": "Title",
+          "embedded": true
+        }
+      },
+      "$a": {"about": "_:title", "property": "mainTitle"},
+      "$d": {"addProperty": "legalDate"},
+      "$f": {"property": "originDate"},
+      "$g": {"link": "genreForm", "resourceType": "GenreForm", "property": "label"},
+      "$h": {"ignored": true},
+      "$k": {"addProperty": "natureOfContent"},
```

Comment:
$k differs also from the mapping of pattern titledetails, used by all other X30 fields. It would be a good idea to create an issue to change mapping of $k for all other X30 fields. Also includes global changes of existing data.